### DISCUSSION TITLE
golangci-lint built from unknown

### DISCUSSION THREAD

**Discussion by BigBoulard** [golangci/golangci-lint#3547] [Q&A] [answered]
Hi guys,

I'm trying to update `golangci-lint` but I can't remember how it's been installed: brew, source.. maybe vscode, included in golang pkg? (Let me know if I say something stupid here)

When I type `golangci-lint --version` I get the following:

```shell
golangci-lint --version                  
golangci-lint has version v1.49.0 built from (unknown, mod sum: "h1:I8WHOavragDttlLHtSraHn/h39C+R60bEQ5NoGcHQr8=") on (unknown)
```

If I try to install using brew, it looks ok at first blush:

```shell
Summary 🍺 /usr/local/Cellar/golangci-lint/1.50.1_1: 9 files, 30MB
```

... but once I type `golangci-lint --version` I can see that it still 1.49 so I guess there 2 installs or something like this.

Thanks for your help

**Comment by ldez** [comment, marked as answer]
Hello,

check our installation page maybe you will remember how you install it.

https://golangci-lint.run/usage/install/

**Reply by BigBoulard** [reply]
Thank you but I already checked that page and unfortunately I can't remember

**Reply by ldez** [reply]
you can use the following command to find the real place:
```console
$ type golangci-lint                                       
golangci-lint is /usr/bin/golangci-lint
```

**Reply by BigBoulard** [reply]
It's in `/Users/me/go/bin/golangci-lint`. I don't know if it's included in the `go` pkg, neither if VSCode can install it too btw

**Reply by ldez** [reply]
so you can just remove this binary and use your preferred method to install the new version.

**Reply by BigBoulard** [reply]
`rm -rf /Users/me/go/bin/golangci-lint` and start the procedure over again, right?

**Reply by ldez** [reply]
yes `rm /Users/me/go/bin/golangci-lint` and maybe uninstall it from brew if the brew installation is not what you expect.

**Reply by BigBoulard** [reply]
Thank you so much for your help. Appreciate it :)

```
golangci-lint --version
golangci-lint has version 1.50.1 built from 8926a95 on 2022-10-22T10:48:48Z
```
